### DISCUSSION TITLE
BsonIgnore on MongODM

### DISCUSSION THREAD

**Discussion by paring-chan** [Etherna/mongodm#59] [Q&A] [answered]
I'm adding additional properties that should not be included in the database. But I don't know how to exclude them.

**Comment by tmm360** [comment, marked as answer]
Hi @pikokr, currently model mapping is based on official MongoDB driver's `BsonClassMap` https://mongodb.github.io/mongo-csharp-driver/2.17/reference/bson/mapping/.
It will change with next major, but I still don't have an ETA.
If you still have issues, a code snippet could help.

**Reply by paring-chan** [reply]
I set `JsonIgnore` instead of `BsonIgnore`(my mistake) and that was why it didn't work. Thank you!
